core/kernel/sprintf_spec.rb

```diff
@@ -3,6 +3,14 @@
 require_relative 'shared/sprintf'
 require_relative 'shared/sprintf_encoding'
 
+describe :kernel_sprintf_to_str, shared: true do
+  it "converts the format_string using #to_str" do
```

Comment:
**suggestion:** I would be more precise e.g. with:
```suggestion
  it "converts the format string argument to String with #to_str when passed not String" do
```

Reply:
https://github.com/ruby/spec/blob/ee20cf35d2d6ce4f1843d74136227c817487f479/core/io/printf_spec.rb#L16

I used this existing line instead of your recommendation, to keep things consistent. If we decide to use a different description, we should probably update it in `core/io/printf_spec.rb` as well.